common/datasets/librispeech/cart.py

```diff
@@ -0,0 +1,939 @@
+"""
+Contains the prepared CART questions for LibriSpeech,
+one variant including the ARPAbet Stress-Markers, the other not.
+"""
+__all__ = [
+    "CartQuestionsWithoutStress",
+    "CartQuestionsWithStress",
+]
+
+
+class CartQuestionsWithoutStress:
+    def __init__(self, max_leaves=9001, min_obs=1000, add_unknown: bool = True):
```

Comment:
would we not have `max_leaves=12001` as default?

Reply:
hm to make it consistent with the baseline yes..